cob_bringup/components/cam3d_zr300_rgbd.launch

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<launch>
+	<arg name="robot"/>
+	<arg name="name"/>
+	<arg name="num_worker_threads" default="4"/>
+	<arg name="start_manager" default="true"/>
+	<arg name="flip" default="true"/>
+	<arg name="sim" default="false"/>
+	<arg name="serial_no" default=""/>
+
+	<include unless="$(arg sim)" file="$(find realsense_camera)/launch/zr300_nodelet_rgbd.launch">
+		<arg name="camera" value="$(arg name)"/>
+		<arg name="num_worker_threads" value="$(arg num_worker_threads)" />
+		<arg name="serial_no" value="$(arg serial_no)"/>
+	</include>
+	<include file="$(find cob_bringup)/tools/hz_monitor.launch">
+		<arg name="robot" value="$(arg robot)"/>
+		<arg name="yaml_name" value="$(arg name)"/>
+		<arg name="sim" value="$(arg sim)"/>
+	</include>
+	<include if="$(arg flip)" file="$(find cob_bringup)/drivers/image_flip_nodelet.launch">
+		<arg name="robot" value="$(arg robot)"/>
+		<arg name="camera_name" value="$(arg name)"/>
+		<arg name="nodelet_manager" value="$(arg name)_nodelet_manager"/>
+		<arg name="start_manager" value="$(arg start_manager)"/>
+	</include>
+
+
+	<group ns="$(arg name)" if="$(arg sim)">
+		<!-- static frames broadcasted by camera driver -->
```

Comment:
values are correct for zr300 as well
still frame_ids are wrong (not prefixed correctly by driver see https://github.com/intel-ros/realsense/issues/258